### PR TITLE
(maint) Add support for Cumulus

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: configs/platforms/cumulus-2.2-amd64.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+platform "cumulus-22-amd64" do |plat|
+  plat.servicedir "/etc/init.d"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "sysv"
+  plat.codename "cumulus"
+
+  plat.apt_repo "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-wheezy.deb"
+
+  plat.provision_with %Q{
+echo 'deb http://enterprise.delivery.puppetlabs.net/build-tools/debian/CumulusLinux CumulusLinux-2.2 build-tools
```
Comment (stahnma) [inline configs/platforms/cumulus-2.2-amd64.rb]: Why can't these be plat.apt_repo ?

Comment (geoffnichols) [inline configs/platforms/cumulus-2.2-amd64.rb]: I think there was some discussion of this in https://github.com/puppetlabs/puppet-agent/pull/66. I don't know if the same constraint still exists today.

Comment (haus) [inline configs/platforms/cumulus-2.2-amd64.rb]: i don't see cumulus in the pooler.

**Comment by haus** [comment]
:+1:

**Pull request merged**